File: src/app/dataModels/tyrantsOfTheUnderdarkSaveModel.ts
import { TyrantsOfTheUnderdarkMatchModel } from "./tyrantsOfTheUnderdarkMatchModel";

export interface TyrantsOfTheUnderdarkSaveModel extends TyrantsOfTheUnderdarkMatchModel {
    matchId: string;
    dateMatch: Date;
    totalCount: number;
}

File: BoardGames.Domain/DataModels/TyrantsOfTheUnderdarkSaveModel.cs
namespace BoardGames.Domain.DataModels;

public class TyrantsOfTheUnderdarkSaveModel : TyrantsOfTheUnderdarkMatchModel
{
    public Guid MatchId { get; set; }

    public DateTime DateMatch { get; set; }

    public int TotalCount { get; set; }
}